packages/react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -81,6 +83,8 @@ export interface TooltipProps
 /**
  * A tooltip that displays information related to an element when the element receives keyboard focus
  * or the mouse hovers over it.
+ * @accessibility
```

Comment:
Should we add example for this one on the docs?

Reply:
I was thinking about it. From one side this behavior changes one attribute and it is listed in the `accessibility` section on cases when to use it. And at other side, it is a variance of a tooltip.
If you think better to add it, I'll do that

Reply:
@mnajdova I added a tooltip best practices section and mentioned it there. I think should be enough for now. Let's observe, if discoverability is bad, will add a separate example in future